PullToRefreshDemo/BeatAnimator.swift

```diff
@@ -28,52 +28,52 @@ import QuartzCore
 
 class BeatAnimator: UIView, PullToRefreshViewDelegate {
     
-    private let layerLoader = CAShapeLayer()
-    private let layerSeparator = CAShapeLayer()
+    fileprivate let layerLoader = CAShapeLayer()
+    fileprivate let layerSeparator = CAShapeLayer()
```

Comment:
These are, but some others aren't. It is simple change I think so I would prefer it to have right.